THE-app/app/(tabs)/booksScreen.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { ScrollView, StyleSheet } from 'react-native';
+import { Text, View } from '@/components/Themed';
+import BookCard from '@/components/BookCard';
+import { Book } from '@/types/Book';
+
+import { gql, useQuery } from '@apollo/client';
+
+const GET_BOOKS = gql`
+  query {
+    viewer {
+      books {
+        hits {
+          id
+          displayTitle
+          url
+          description
+          subjects {
+            name
+          }
+          levels {
+            name
+          }
+          valid
+        }
+      }
+    }
+  }
+`;
+
+function BooksScreen() {
+  const { loading, error, data } = useQuery(GET_BOOKS);
+
+  if (loading) return <Text>Loading...</Text>;
+  if (error) return <Text>Error :( {error.message}</Text>;
+
+  const books: Book[] = data.viewer.books.hits.filter((book: Book) => book.valid);
+
+  return (
+    <ScrollView contentContainerStyle={styles.container}>
+      {books.map((book: Book) => (
```

Comment:
You probably don't need to type the lambda here. TS type inference will do it for you. Using explicit types can be counter-productive because it makes it harder to refactor your code.